src/pages/en/guides/environment-variables.md

```diff
@@ -17,23 +15,18 @@ See the official [Environment Variables example](https://github.com/withastro/as
 SECRET_PASSWORD=password123
 PUBLIC_ANYBODY=there
 ```
-<p>
-In this example, <code>PUBLIC_ANYBODY</code> (accessible via <ImportMetaEnv path=".PUBLIC_ANYBODY" />) will be available in server or client code, while <code>SECRET_PASSWORD</code> (accessible via <ImportMetaEnv path=".SECRET_PASSWORD" />) will be server-side only.
-</p>
+
+In this example, `PUBLIC_ANYBODY` (accessible via `import.meta.env.PUBLIC_ANYBODY`) will be available in server or client code, while `SECRET_PASSWORD` (accessible via `import.meta.env.SECRET_PASSWORD`) will be server-side only.
 
 ## Default environment Variables
 
 Astro includes a few environment variables out-of-the-box:
-<ul>
-<li> <ImportMetaEnv path=".MODE" /> (<code>development</code> | <code>production</code>): the mode your site is running in. This is <code>development</code> when running <code>astro dev</code> and <code>production</code> when running <code>astro build</code>.</li>
-
-<li> <ImportMetaEnv path=".BASE_URL" /> (<code>string</code>): the base url your site is being served from. This is determined by the <a href="/en/reference/configuration-reference/#base"><code>base</code> config option</a>.</li>
 
-<li> <ImportMetaEnv path=".PROD" /> (<code>boolean</code>): whether your site is running in production.</li>
-
-<li> <ImportMetaEnv path=".DEV" /> (<code>boolean</code>): whether your site is running in development (always the opposite of <ImportMetaEnv path=".PROD" />).</li>
-<li><ImportMetaEnv path=".SITE" /> (<code>string</code>): <a href="/en/reference/configuration-reference/#site">The <code>site</code> option</a> specified in your project's <code>astro.config</code>.</li>
-</ul>
+- `import.meta.env.MODE` (`development` | `production`): the mode your site is running in. This is `development` when running `astro dev` and `production` when running `astro build`.
+- `import.meta.env.BASE_URL` (`string`): the base url your site is being served from. This is determined by the <a href="/en/reference/configuration-reference/#base">`base` config option</a>.
```

Comment:
Just checking @hippotastic , does this (and the next link in this section) still need to be in `<a>` tags, or would markdown link format work here? This would seem more consistent with the other changes removing `<li>` etc. (Applies to all languages.)

Reply:
Good eye! I guess Markdown links should work well here. I'll fix this across all languages, too.

Reply:
Done! Thanks @sarah11918 :)